enterprise-suite/gotests/testenv/testenv.go

```diff
@@ -62,24 +63,14 @@ func InitEnv() {
 		Expect(err).To(Succeed(), "new k8sclient")
 	}
 
-	var lbcArgs []string
-	helmArgs := []string{"--set esConsoleURL=http://console.test.bogus:30080"}
+	// For test set-up installation, ignore any PVC warnings by setting --delete-pvcs.
+	lbcArgs := []string{"--delete-pvcs"}
+
+	// Always enable persistent volumes to improve our test coverage with this important configuration.
+	// This should work in all known standard clusters (except for Minishift, which doesn't provide a default StorageClass).
+	helmArgs := []string{"--set esConsoleURL=http://console.test.bogus:30080", "--set usePersistentVolumes=true"}
 	if isMinikube {
 		helmArgs = append(helmArgs, "--set exposeServices=NodePort")
-	} else {
-		// Look for expected storage classes, run with emptyDir if they don't exist
-		var foundStorageClass bool
-		for _, storageClass := range []string{"standard", "gp2"} {
-			if kube.StorageClassExists(K8sClient, storageClass) {
-				foundStorageClass = true
-				helmArgs = append(helmArgs,
-					fmt.Sprintf("--set usePersistentVolumes=true,defaultStorageClass=%v", storageClass))
-				break
-			}
-		}
-		if !foundStorageClass {
-			helmArgs = append(helmArgs, "--set usePersistentVolumes=false")
-		}
```

Comment:
I don't think this is necessary now that we have support the default storage class.

Reply:
It will no longer work on minishift without default storage class, but I think that's acceptable. Should we even have an option of running console with `emptyDir`? It could slightly simplify helm charts not to have it at all.

Reply:
Do we need to support minishift with gotests? I didn't think so.

Reply:
I think we should still support emptyDir "in case". It's not a huge burden.